client/src/components/Column/Column.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import useStyles from './useStyles';
+import Card from '../Card/Card';
+import { Droppable, Draggable } from 'react-beautiful-dnd';
+import { Grid } from '@material-ui/core';
+import { MoreHoriz } from '@material-ui/icons';
+export default function Column(props: any): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Draggable draggableId={props.column.id} index={props.index}>
+      {(provided) => (
+        <div className={classes.container} {...provided.draggableProps} ref={provided.innerRef}>
```

Comment:
Please use MUI `Box` component instead of using a `div`

Reply:
so I tried changing the div into Box component but ts is giving error
`Property 'ref' does not exist on type 'IntrinsicAttributes & BoxProps & { children?: ReactNode; }'`
I can update add the ref prop to Box but that would mean update it everytime. 
They fixed this issue on v5 but we are using v4. any suggestions?